doc/source/building/index.rst

```diff
@@ -16,4 +16,5 @@ Build instructions for different operating systems:
    linux
    windows
    macosx
-
+   
+For additional information, see the :ref:`build-install-faq`.
```

Comment:
Links to [Build/Install FAQ](https://github.com/scipy/scipy/pull/10286/files#diff-ff7a395fc10e317f57cd03639d848aeb)